k8s-opensearch/_providers.tf

```diff
@@ -7,5 +7,8 @@ terraform {
     aws = {
       source  = "hashicorp/aws"
     }
+    tls = {
+      source  = "hashicorp/tls"
```

Comment:
my testing for this was to add a `version = "= 3.1.0"` here and run `terraform validate` on this module with the added field in the other file. I added this source reference just as best practice / for consistency.

Reply:
👍 

Sorry if this is what you were asking earlier. I think we _do_ want the version here, my point was that if it doesn't include the version in cached-providers then it will fail. Adding the version here enables local dev that will match our execution environment.

Reply:
@WillBeebe IMHO we should add something to our CLI to `mass bundle init` to run tf init but grab provider versions from that centralized place. otherwise we're gonna be maintaining all those versions in many places and drift is inevitable.